### DISCUSSION TITLE
vSomeIP 3.5.1 and Network Tests

### DISCUSSION THREAD

**Discussion by duartenfonseca** [COVESA/vsomeip#784] [Announcements]
The **3.5.1 version** was released, and we encourage you to test it, check if some of your bugs could be solved by this new version.
We also added a new way of running the **network tests** (https://github.com/COVESA/vsomeip/wiki/How-to-run-network-tests) . It would be important to start using this tool for new Pull Requests that you want to share with the community.
The feedback regarding this tool is also important, so that we can check if you can run this locally

### **UPDATE:**
this version was found to have introduced a bug, related to two commits: (https://github.com/COVESA/vsomeip/commit/f55505cd93238ae70edc66003e0d01800f026c27) (https://github.com/COVESA/vsomeip/commit/0ab85df10f7eb15b1530010217eac55f343efef7)
A new version is being created that reverts this changes.
For now you should not use this version, or keep it and revert the changes yourself.

**Comment by vidz12** [comment]
When is the next stable version expected?

**Reply by goncaloalmeida** [reply]
@vidz12: https://github.com/COVESA/vsomeip/discussions/820
